README.md

```diff
@@ -144,12 +161,91 @@ Include content from "Intro" up to the next heading of the same level:
 `inline`
 :   Flag that tells the preprocessor to replace sequences of whitespace characters of many kinds (including `\r`, `\n`, and `\t`) with single spaces (` `) in the included content, and then to strip leading and trailing spaces. It may be useful in single-line table cells. Default value is `false`.
 
+`project_root`
+:   Path to the top-level (“root”) directory of Foliant project that the included file belongs to. This option can be necessary to resolve correctly the `!path` and the `!project_path` modifiers in the included content.
```

Comment:
:   Path to the top-level (“root”) directory of Foliant project that the included file belongs to. This option may be needed to resolve `!path` and the `!project_path` modifiers in the included content properly.